Monitoring/prometheus.yml

```
@@ -0,0 +1,14 @@
+global:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)